<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendDataToTcp</class>
 <widget class="QMainWindow" name="SendDataToTcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SendDataToTcp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>本地IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="localIP">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>本地端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="localPort">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>时间/s</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>目的IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="servicerIP">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>目的端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="servicerPort">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="OpenFileBt">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>打开文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendBt">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>发送数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeBt">
          <property name="text">
           <string>取消发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
